### PR TITLE
L10n fr

### PR DESCRIPTION
**Pull request by tdd**
Hey again Domenic,

As mentioned in #27, this is the second step of our recent work on globalizing and localizing `count-to-6`. This includes the commits referenced in #27, which will "disappear" when/if you merge that one first, to leave only FR-related commits for actual localization.

I remain, naturally, available for any discussion on this.

Looking forward to our getting that in,

Best,

### CODE REVIEW

**Comment by tdd** [comment]
Hey @domenic,

So, due to your feedback including lots of tweaks on commit messages, I had to rebase and force-push the entire branch, which means a number of your comments were lost in this PR (many are still visible in #27, based on the former early commits, though).

About wrapping: please consider my notes in the new, long-form commit message of c064433. If you're still adamant on no-wrapping, I'll rebase again and dewrap it, but I do feel it'd end up less usable for readers. Still, your house, your rules :wink: 

Cheers,

**Comment by tdd** [comment]
Hey @domenic all done! I hope this meets your approval.

File: lib/exercises/arrow_functions_and_this/problem.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 Arrow functions help to make the confounding issue of `this` less confounding.
 
-Arrow functions lexically bind the value of `this`. It is very likely that you do not know what that sentence means. And that's OK.
+Arrow functions lexically bind the value of `this`.  It is very likely that you do not know what that sentence means. And that's OK.
```
Comment (domenic) [inline lib/exercises/arrow_functions_and_this/problem.md]: No double-spaces please

File: lib/exercises/arrow_functions_and_this/problem.fr.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
+Les fonctions fléchées nous aident aussi à mieux maîtriser la valeur de `this`,
```
Comment (domenic) [inline lib/exercises/arrow_functions_and_this/problem.fr.md]: Can you remove the wrapping in these as well?

File: lib/exercises/arrow_functions_and_this/problem.md (hunk 1 above)
Comment (tdd) [inline lib/exercises/arrow_functions_and_this/problem.md]: Really?  I find them to be **so** much more readable in monospace fonts, clearly delimiting sentences.

Ah well, then again, your house…  Will do in the morning, along with the wrap fixes I missed.

**Comment by tdd** [comment]
Hey @domenic, I believe these two latest commits now address all your remaining concerns.

Best,

**Comment by domenic** [comment]
Thanks! Merged as 33da9f4b4a57ed5032f8f1dc38f778307f1a8d64 and released as 0.7.0.

**Comment by tdd** [comment]
Awesome :tada:

-- 
Christophe Porteneuve
tdd@tddsworld.com

**Pull request closed without merging**